lua/neogit/config.lua

```diff
@@ -1,157 +1,493 @@
 local M = {}
 
-function M.get_reversed_status_maps()
-  return vim.tbl_add_reverse_lookup(vim.tbl_deep_extend("force", M.values.mappings.status, {}))
+function M.get_copy()
+  return vim.tbl_deep_extend("force", M.values, {})
 end
 
-M.values = {
-  disable_hint = false,
-  disable_context_highlighting = false,
-  disable_signs = false,
-  disable_commit_confirmation = false,
-  disable_builtin_notifications = false,
-  telescope_sorter = function()
-    return nil
-  end,
-  disable_insert_on_commit = true,
-  use_per_project_settings = true,
-  remember_settings = true,
-  use_magit_keybindings = false,
-  auto_refresh = true,
-  sort_branches = "-committerdate",
-  kind = "tab",
-  -- The time after which an output console is shown for slow running commands
-  console_timeout = 2000,
-  -- Automatically show console if a command takes more than console_timeout milliseconds
-  auto_show_console = true,
-  status = {
-    recent_commit_count = 10,
-  },
-  commit_editor = {
-    kind = "split",
-  },
-  commit_select_view = {
-    kind = "tab",
-  },
-  commit_view = {
-    kind = "vsplit",
-  },
-  log_view = {
-    kind = "tab",
-  },
-  rebase_editor = {
-    kind = "split",
-  },
-  reflog_view = {
-    kind = "tab",
-  },
-  merge_editor = {
-    kind = "split",
-  },
-  preview_buffer = {
-    kind = "split",
-  },
-  popup = {
-    kind = "split",
-  },
-  signs = {
-    hunk = { "", "" },
-    item = { ">", "v" },
-    section = { ">", "v" },
-  },
-  integrations = setmetatable({}, {
-    __index = function(_, key)
-      local ok, value = pcall(require, key)
-      return ok and value or false
+function M.get_default_values()
+  return {
+    disable_hint = false,
+    disable_context_highlighting = false,
+    disable_signs = false,
+    disable_commit_confirmation = false,
+    disable_builtin_notifications = false,
+    use_telescope = false,
```

Comment:
`use_telescope` - This line got removed earlier today

Reply:
Removed from my branch now.